bin/hdfcoinc/pycbc_coinc_mergetrigs

```diff
@@ -107,8 +106,11 @@ for filename in args.trigger_files:
                 if not gk in gating:
                     gating[gk] = numpy.array([], dtype=numpy.float64)
                 gating[gk] = numpy.append(gating[gk], gk_data[:])
-    data.close()    
-f['%s/search/start_time' % ifo], f['%s/search/end_time' % ifo] = start, end
+    data.close()
+
+segments = numpy.array(sorted(segments), dtype=numpy.float64)
```

Comment:
is it clear what sorting will do to 'segments' (a set of tuples) ? is it desirable?

Reply:
It will convert the set to a list [sorted by the first item of each tuple](https://docs.python.org/3/reference/expressions.html#value-comparisons), i.e. the start time of the segment, which seemed desirable to me.

Reply:
Added a comment with the intended behavior for good measure…